set.mm

```diff
@@ -555015,7 +555007,8 @@ relation with an intersection with the same Cartesian product (cf.
       UIUNTTEUEUBUFUGUH $.
   $}
 
-  $( Lemma for ~ ineccnvmo2 .  (Contributed by Peter Mazsa, 3-Sep-2021.) $)
+  $( Equivalence of an "at most one" and a restricted "at most one" inside a
+     universal quantification.  (Contributed by Peter Mazsa, 3-Sep-2021.) $)
   alrmomo $p |- ( A. x E* y e. ran R x R y <-> A. x E* y x R y ) $=
     ( cv wbr crn wrmo wmo wcel wa df-rmo ccnv cres cnvresrn breqi cvv brresALTV
```

Comment:
This is not a general theorem about an "at most one" and a restricted "at most one" inside a universal quantification, but for a range. I would call it "alrmomorn" (and adjust the comment accordingly).

Reply:
@avekens perhaps this way. If we want to be more specific, we should consider to relabel and comment them as lemmas.

Reply:
It's fine now.